HackerRank Problem/Python/Pairs.py

```diff
@@ -0,0 +1,32 @@
+import math
+import os
+import random
+import re
+import sys
+
+def pairs(k, arr):
+    d = {}
+    answer = 0
+    for i in arr:
+        d[i] = i
+    for j in arr:
+        g = j+k
+        if g in d:
+            answer +=1
+    return answer
+if __name__ == '__main__':
+    fptr = open(os.environ['OUTPUT_PATH'], 'w')
+
+    nk = raw_input().split()
+
+    n = int(nk[0])
+
+    k = int(nk[1])
+
+    arr = map(int, raw_input().rstrip().split())
```

Comment:
you are not getting my point. After taking the inputs like nk, n, k, arr there are extra lines. remove those only